README.md

```diff
@@ -1,33 +1,174 @@
+One could add information accordingly in the different fields. The information would be automatically updated after successful push.
+
+### Blog
+
+A link to blog posts can be found in the navigation menue at the top of the website. 
+The Docusaurus blog feature is described in detail in the [Docusaurus documentation](https://docusaurus.io/docs/blog). 
+By default when someone clicks on `blog` link they will be directed to a page containing list of all present blogs on the website.
+
+Steps to add a blog post:
+
+1. Create a new file named  `blog/yyyy-mm-dd-title.md`
+2. In the top of your blog file add a yaml header.
+3. Add at least `title`, `author` and `tags` yaml fields. 
+  - These metadata fields help with search and navigation on the site. 
+  - These and other yaml header options are described in the [blog header options](https://docusaurus.io/docs/blog#header-options) section of the Docusaurus documentation.   
+4. writing your blog and save the file. 
+5. any images can be put in `static/img/` and referenced using a relative path `static/img/myimage.png` 
+6. build the website to make sure the blog shows up as expected 
+7. After the new changes are merged into the main branch, the new blog post will appear on the website.
+
+![YamlHeader](screenshots/yamlHeader.png)
+
+<!-- instead of that screenshot, perhaps provide a code chunk here  -->
+
+To learn more about markdown visit (https://guides.github.com/features/mastering-markdown/). 
+
+
+## Screenshots
+
+This section contains screenshots of website's homepage in light and dark theme.
+
+![LightTheme](screenshots/light.png)
+
+![DarkTheme](screenshots/dark.png)
+
+## Contributing
+
+This is an open source project and contributions are what make an open source community.
+Everyone is welcome to contribute to this project. The rules are simple:
+
+- Be polite, be patient.
+- Be specific about your doubts, make sure you read this documentation carefully.
+
+First time contributors are welcome. Contributions can be very simple, make sure to check out beginner friendly issues, if present.
+
+## Next Steps
+
+This section contains future plans for the website. New contributors can use this section as a reference for planning their contributions.
+
+### Optimising Documentation
+
+At present the PEcAn's documentation is being served from (https://pecanproject.github.io/pecan-documentation/master/) using an iframe. While this saves a lot of overhead in converting Rmd files in (https://github.com/PecanProject/pecan/tree/develop/book_source) to md files so that docusaurus could render them, it also increases the loading time for documentation, it also looks less consistent.
+In future, it is suggested to find an optimised way to convert rmd files to md and integrate them via docusaurus's native documentation parser.
```

Comment:
maybe create a new issue with details, and refer to it. But this should include next steps / outline of what needs to be done e.g. 

- automating build to md rather than html (not natively supported by bookdown)
- what are steps to integrate ... 
  - build md files to (what folder?)
  - change config files to ???
  - remove the current iframe approach (where?)

Reply:
Got it, adding a new issue is a good idea.